CompilerSource/compiler/components/parse_and_link.cpp

```diff
@@ -95,7 +95,7 @@ int lang_CPP::compile_parseAndLink(EnigmaStruct *es,parsed_script *scripts[], ve
       // Keep a parsed record of this timeline
       tline_lookup[es->timelines[i].name].push_back(tlines.back());
       parser_main(newcode, &tlines.back()->pev, script_names);
-      edbg << "Parsed `" << es->timelines[i].name <<", moment: " <<es->timelines[i].moments[j].stepNo << "': " << tlines.back()->obj.locals.size() << " locals, " << tlines.back()->obj.globals.size() << " globals" << flushl;
+      edbg << "Parsed `" << es->timelines[i].name <<", moment: " <<es->timelines[i].moments[j].stepNo << "': " << (int)(tlines.back()->obj.locals.size()) << " locals, " << (int)tlines.back()->obj.globals.size() << " globals" << flushl;
```

Comment:
I am hoping this file can be rolled back, now, too. If not, please let me know and I will make sure that it can.